src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -106,6 +110,8 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit {
 
   public isActive: boolean = false;
 
+  public hasItemContent: boolean = false;
```

Comment:
I tested this locally, but if I dynamically add a `SkyRepeaterItemContentComponent` after a few seconds, the repeater's chevron isn't reapplied. Maybe you can subscribe to the `this.repeaterItemContent.changes` observable and rerun the `hasItemContent` check?

Reply:
Good call, I updated it to use ContentChildren and check for one initially and then subscribe to changes on that and update the value on changes as well.

Reply:
@blackbaud-GavinNicol We're still seeing the same behavior. Here's an example:
```
    <sky-repeater-item>
      <sky-repeater-item-title>
        Title
      </sky-repeater-item-title>
      <sky-repeater-item-content *ngIf="isVisible">
        Content
      </sky-repeater-item-content>
    </sky-repeater-item>

...

  <button
    (click)="isVisible = true"
    type="button"
  >
    Show repeater item content
  </button>
```

If you click the button, the repeater item shows up, but the chevron is still hidden. I'm wondering if the `isCollapsible` boolean isn't getting updated?

Reply:
Alright, got it working now. it was that `isCollapsible` was not being updated correctly.

Reply:
Yep, works for me. Thank you!